pkg/interceptors/interceptors.go

```diff
@@ -159,7 +162,28 @@ func ResolveToURL(getter InterceptorGetter, name string) (*apis.URL, error) {
 	return ic.ResolveAddress()
 }
 
-func Execute(ctx context.Context, client *http.Client, req *triggersv1beta1.InterceptorRequest, url string) (*triggersv1beta1.InterceptorResponse, error) {
+func Execute(ctx context.Context, client *http.Client, req *triggersv1beta1.InterceptorRequest, url string,
+	certData []byte) (*triggersv1beta1.InterceptorResponse, error) {
+
+	// There is a chances that url may come from ClusterInterceptorSpec/ClientConfig of URL or Service
+	// So adding check to make sure its https before creating client for https.
+	if strings.Contains(url, "https") {
+		certPool := x509.NewCertPool()
+		if ok := certPool.AppendCertsFromPEM(certData); !ok {
+			return nil, fmt.Errorf("unable to parse cert from %s", certData)
+		}
+
+		client = &http.Client{
+			Transport: &http.Transport{
+				// disable "G402 (CWE-295): TLS MinVersion too low. (Confidence: HIGH, Severity: HIGH)"
```

Comment:
@dibyom 
This is the build issue
```
pkg/interceptors/interceptors.go:182:23: G402: TLS MinVersion too low. (gosec)
				TLSClientConfig: &tls.Config{
					RootCAs: certPool,
				},
```

Reply:
@dibyom added [MinVersion](https://github.com/tektoncd/triggers/pull/1333/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-706a824f91e384361f83950baa4b8624bd38da1ec63805d8c15ec8037502a931R184) to avoid build failure
Ref: https://golang.hotexamples.com/examples/crypto.tls/Config/MinVersion/golang-config-minversion-method-examples.html

PTAL
Thank you